src/main/resources/static/products.js

```diff
@@ -1,18 +1,64 @@
-angular.module('Group-V_Store').controller('ProductsController',
-	['$scope', '$http', '$routeParams', '$location', function($scope, $http, $routeParams, $location) {
-		const type = $routeParams.type || "";
-		const brand = $routeParams.brand || "";
-		
-		$http.get(`http://localhost:8080/products?type=${type}&brand=${brand}`)
-			.then(function(response) {
-				$scope.products = response.data;
+angular.module('Group-V_Store').controller('ProductsController', function($scope, $http) {
+	const queryString = window.location.search;
+	const urlParams = new URLSearchParams(queryString);
+	const type = urlParams.get('type') ?? '';
+	const brand = urlParams.get('brand') ?? '';
+
+	$http.get(`http://localhost:8080/products?type=${type}&brand=${brand}`)
```

Comment:
can you get rid of the localhost here (so that it works when deployed) and just make it `/products?type=${type}&brand=${brand}`

Reply:
done in latest commit